### ISSUE TITLE
simple_decoder_fn_train no longer exists

### ISSUE THREAD

**Issue by ghost** [udacity/dlnd-issue-reports#252]
In the Translation project, there is a code block (Decoding - Training) that tells students to use this function to create logits.  It is no longer in Tensorflow. I've tried versions 0.12 and 1.0.1 on Windows 10.

**Comment by Brok-Bucholtz** [comment]
The [`tf.contrib.seq2seq.simple_decoder_fn_train`](https://www.tensorflow.org/api_docs/python/tf/contrib/seq2seq/simple_decoder_fn_train) function is only supported in 1.0 and 1.0.1.  You might have installed 1.1.
